db/seeds.rb

```diff
@@ -1,88 +1,2 @@
-    {:component => "text-field", :name => "role", :type => "hidden", :initialValue => "ansible"}, # FIXME: Find the correct value.
-    {:component => "text-field", :name => "authtype", :type => "hidden", :initialValue => "username_password"},
-    {:component => "text-field", :name => "url", :label => "URL"},
-    {:component => "checkbox", :name => "verify_ssl", :label => "Verify SSL"},
-    {:component => "text-field", :name => "certificate_authority", :label => "Certificate Authority", :condition => {:when => "verify_ssl", :is => true}},
-    {:component => "text-field", :name => "username", :label => "User name"},
-    {:component => "text-field", :name => "password", :label => "Secret Key", :type => "password"}
-  ]
-}
-update_or_create(SourceType, :name => "ansible-tower", :product_name => "Ansible Tower", :vendor => "Red Hat", :schema => ansible_tower_json_schema)
-update_or_create(SourceType, :name => "vsphere", :product_name => "VMware vSphere", :vendor => "VMware")
-update_or_create(SourceType, :name => "ovirt", :product_name => "Red Hat Virtualization", :vendor => "Red Hat")
-update_or_create(SourceType, :name => "openstack", :product_name => "Red Hat OpenStack", :vendor => "Red Hat")
-update_or_create(SourceType, :name => "cloudforms", :product_name => "Red Hat CloudForms", :vendor => "Red Hat")
-
-update_or_create(ApplicationType,
-                 :name                           => "/insights/platform/catalog",
-                 :display_name                   => "Catalog",
-                 :dependent_applications         => ["/insights/platform/topological-inventory"],
-                 :supported_source_types         => ["ansible_tower"],
-                 :supported_authentication_types => {"ansible_tower" => ["username_password"]})
-
-update_or_create(ApplicationType,
-                 :name                           => "/insights/platform/cost-management",
-                 :display_name                   => "Cost Management",
-                 :dependent_applications         => [],
-                 :supported_source_types         => ["amazon"],
-                 :supported_authentication_types => {"amazon" => ["arn"]})
-
-update_or_create(ApplicationType,
-                 :name                           => "/insights/platform/topological-inventory",
-                 :display_name                   => "Topological Inventory",
-                 :dependent_applications         => [],
-                 :supported_source_types         => ["amazon", "ansible_tower", "azure", "openshift"],
-                 :supported_authentication_types => {
-                   "amazon"        => ["access_key_secret_key"],
-                   "ansible_tower" => ["username_password"],
-                   "azure"         => ["username_password"],
-                   "openshift"     => ["token"]
-                 })
+# Run all seed files in db/seeds/*.rb
+Dir[File.join(Rails.root.join("db", "seeds"), "**/*.rb")].each { |f| load f }
```

Comment:
Does `Dir[Rails.root.join("db", "seeds", "**", "*.rb")].each { |f| load f }` work?

Reply:
Should we `require` rather than `load`?

Reply:
> Does Dir[Rails.root.join("db", "seeds", "**", "*.rb")].each { |f| load f } work?

Yes

Reply:
> Should we require rather than load?

:+1: Done